src/components/Details.js

```diff
@@ -0,0 +1,119 @@
+/* eslint-disable camelcase */
```

Comment:
- [ ] Let's avoid disabling linters, instead, we should learn how to fix errors that are shown to have a better app.

Reply:
- [ ] Like the first code reviewer said please avoid disabling the linter and fix the error.